tests/README.md

```diff
@@ -77,17 +77,18 @@ The build script has the following parameters:
 Select `Ubuntu 14.04 LTS v1503 (beta with Docker support)`.
 The platform with *Docker support* means the tests will run in a VM.
 
-## Manually running the functional tests
+## Manually running the tests
 
 Make sure to pass `--enable-functional-tests` to the configure script, then, after building the project, you can run the tests.
```

Comment:
Ok, sorry, i was a little bit confused the the `make unit-check` command is explained in the *functional test* section. What about splitting it into something like that?

> ## Manually running the tests
> 
> The tests can be run manually. There is a rule to run unit, functional and all tests.
> 
> ### Unit tests
> 
> The unit tests can be run with `make unit-check` after you [built](../Documentation/hacking.md#building-rkt) the project.
> 
> ### Functional tests
> 
> The functional tests require to pass `--enable-functional-tests` to the configure script, then, after building the project, you can run the tests.
> 
> ```
> ./autogen.sh
> ./configure --enable-functional-tests
> make -j4
> make functional-check
> ```
> 
> For more details about the `--enable-functional-tests` parameter, see [configure script parameters documentation](../Documentation/build-configure.md#--enable-functional-tests).
> 
> ### All tests
> 
> To run all tests, see [functional tests](./README.md#functional-tests) to configure and build it with functional tests enabled. Instead of `make functional-check` you have to call `make check` to run all tests.
> 
> ### Passing additional parameters
> 
> You can use a `GO_TEST_FUNC_ARGS` variable to pass additional parameters to `go test`.
> This is mostly useful for running only the selected functional tests.
> The variable is ignored in unit tests.
> 
> ```
> make check GO_TEST_FUNC_ARGS='-run NameOfTheTest'
> make functional-check GO_TEST_FUNC_ARGS='-run NameOfTheTest'
> ```
> 
> Run `go help testflag` to get more informations about possible flags accepted by `go test`.
> 

This is just a proposal, i have no problem to revert it to `functional tests` again :)